README.md

```diff
@@ -337,6 +298,63 @@ a higher priority.
 are more likely to get backwards-incompatible updates. Additionally, it's possible for Alpha
 libraries to get deprecated and deleted before ever being promoted to Beta or GA.
 
+Version Management
+------------------
+
+The easiest way to solve version conflicts is to use google-cloud's BOM. In Maven, add the following to your POM:
+
+[//]: # ({x-version-update-start:google-cloud-bom:released})
+```xml
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bom</artifactId>
+        <version>0.57.0-alpha</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+```
+[//]: # ({x-version-update-end})
```

Comment:
The stuff above just repeats what is in the quick start section, and the stuff below is for version combinations before the BOM. I think I'd prefer to remove the repeated stuff above and rename this section to something like `Old version combinations`. (Also change the text at the end of the quick start that refers to this section.)